docs/advanced/document-store/index.md

```diff
@@ -159,15 +159,15 @@ Creating DocumentArrays without indexes is useful during prototyping but should
 DocArray supports multiple storage backends with different search features. The following table showcases relevant functionalities that are supported (✅) or not supported (❌) in DocArray depending on the backend:
 
 
-| Name                                  | Construction                             |  vector search | vector search + filter | filter        |
-|---------------------------------------|------------------------------------------|----------------|------------------------|---------------|
-| In memory                             | `DocumentArray()`                        | ✅             | ✅                     | ✅            |
-| [`Sqlite`](./sqlite.md)               | `DocumentArray(storage='sqlite')`        | ❌             | ❌                     | ✅            | 
-| [`Weaviate`](./weaviate.md)           | `DocumentArray(storage='weaviate')`      | ✅             | ✅                     | ✅            |
-| [`Qdrant`](./qdrant.md)               | `DocumentArray(storage='qdrant')`        | ✅             | ✅                     | ❌            |
-| [`AnnLite`](./annlite.md)             | `DocumentArray(storage='annlite')`       | ✅             | ✅                     | ✅            |
-| [`ElasticSearch`](./elasticsearch.md) | `DocumentArray(storage='elasticsearch')` | ✅             | ✅                     | ✅            |
-| [`Redis`](./redis.md)                 | `DocumentArray(storage='redis')`         | ✅             | ✅                     | ✅            |
+| Name                                  | Construction                             |  vector search | vector search + filter | filter       |
+|---------------------------------------|------------------------------------------|----------------|------------------------|--------------|
+| In memory                             | `DocumentArray()`                        | ✅             | ✅                     | ✅           |
+| [`Sqlite`](./sqlite.md)               | `DocumentArray(storage='sqlite')`        | ❌             | ❌                     | ✅           | 
```

Comment:
```suggestion
| [`SQLite`](./sqlite.md)               | `DocumentArray(storage='sqlite')`        | ❌             | ❌                     | ✅           | 
```

Reply:
@dongxiang123 I do not see this being applied

Reply:
Updated